### PR TITLE
DROOLS-3814 DMN Submarine Example on Quarkus kie-dmn-rest-utils

### PR DESCRIPTION
**Pull request by tarilabs**
initial commit

Requires https://github.com/kiegroup/drools/pull/2308

### CODE REVIEW

File: drools/kie-dmn/kie-dmn-rest-quarkus/pom.xml [hunk 1]
```diff
@@ -0,0 +1,134 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.kie</groupId>
+    <artifactId>kie-dmn</artifactId>
+    <version>8.0.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>kie-dmn-rest-quarkus</artifactId>
```
Comment (mswiderski) [inline drools/kie-dmn/kie-dmn-rest-quarkus/pom.xml]: our of curiosity why this is named rest-quarkus? I don't see any reference to neither of these

Comment (tarilabs) [reply drools/kie-dmn/kie-dmn-rest-quarkus/pom.xml]: As always, I am always open to naming suggestions :)
I realized after opening the module I only needed JSONB dependency

Comment (tarilabs) [reply drools/kie-dmn/kie-dmn-rest-quarkus/pom.xml]: Renamed to `kie-dmn-rest-utils`
Unless other comments, I believe I can resolve this feedback ;)

**Comment by tarilabs** [comment]
Unless there are other comments, can this be merged, please?
The -example instead will wait.

**Comment by tarilabs** [comment]
( Jenkins is now red since https://github.com/kiegroup/submarine-runtimes/commit/1dd622df55d60cbca070fda4a2d5c59edd7c6ade I believe broke https://github.com/kiegroup/submarine-examples Drools examples )

**Comment by tarilabs** [comment]
Given no further comments where added,
and double-checked https://github.com/kiegroup/submarine-runtimes/pull/26#issuecomment-484257736
I will move to merge this PR.

**Pull request merged**